### ISSUE TITLE
Less alarming error message when last recent directory is not available

### ISSUE THREAD

**Issue by uellue** [LiberTEM/LiberTEM#214]
The file browser tries to open the most recent directory, which is awesome. If I connect the local GUI to different dask clusters, the most recent directory will usually not be available. 

Right now, I am greeted with a large alarming red modal window. But it is not as bad as it looks like: I just have to close it and be on my merry way.

Possible solutions: 
* Don't use red, but a more neutral color when reporting non-existing files or directories. This is in my opinion a rather normal thing, a typo in the path or a changed environment. Right now it looks too alarming.
* If possible, don't report a missing default directory from "most recent" at all. Perhaps just go to the home directory instead?

**Comment by sk1p** [comment]
I agree, we may need to differentiate between warnings and errors. A possible solution would be to have a "log console" kind of thing, where log messages of different severity end up. If possible, even slurp up logging messages from the workers! Then, only display the error modal for "error", "fatal" etc severity messages.

For improving the usability when connecting to different systems, another possibility would be to store the configuration keyed by the current cluster. That way, no `localStorage` configuration is shared anymore, and dataset parameters, last directory etc. should be fine for the most part.

As a key, I wouldn't use the scheduler URI here, because that may be influenced by network redirection magic (i.e. SSH tunnels), but some other identifier. I'd suggest we add a method to the executor that returns this identifier and implement it for dask by executing [`socket.getfqdn()`](https://docs.python.org/3/library/socket.html#socket.getfqdn) or something similar on the scheduler (via [`Client.run_on_scheduler`](https://distributed.dask.org/en/latest/api.html#distributed.Client.run_on_scheduler)).
